py/src/braintrust/oai.py

```diff
@@ -382,68 +378,72 @@ def _parse_params(cls, params: Dict[str, Any]) -> Dict[str, Any]:
 
         # Then, copy the rest of the params
         params = prettify_params(params)
-        input = params.pop("input", None)
+        input_data = params.pop("input", None)
         return merge_dicts(
             ret,
             {
-                "input": input,
+                "input": input_data,
                 "metadata": {**params, "provider": "openai"},
             },
         )
 
+    @classmethod
+    def _parse_event_from_result(cls, result: Dict[str, Any]) -> Dict[str, Any]:
+        """Parse event from response result - minimal processing like JS version."""
+        data = {}
```

Comment:
nit would be nice to be more specific i.e. https://github.com/braintrustdata/braintrust-sdk/blob/64cf4c6299cadfe4e5a7987c8db3383a69a108b9/py/src/braintrust/generated_types.py#L59

in TS we do something like https://github.com/braintrustdata/braintrust-sdk/blob/64cf4c6299cadfe4e5a7987c8db3383a69a108b9/js/util/object.ts#L127-L134